src/main/java/net/fabricmc/loader/api/Loader.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.loader.api;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.loader.FabricLoader;
+
+/**
+ * The public-facing Loader instance.
+ *
+ * TODO: When we get a change to break this:
+ * - remove getMods()List
+ * - move ModInfo to net.fabricmc.api.loader
+ * - add a way to get the ModContainer information, but do not expose
+ *   ModContainer itself! Use another interface or simply separate methods
+ */
+public interface Loader {
```

Comment:
Should this be called Loader or FabricLoader (with the latter FabricLoader being renamed to FabricLoaderImpl in 0.4.0)?

Reply:
FabricLoader might be a better name, Loader doesnt really explain what it is, it could be a loader of anything.